src/js/themes/media-theme-youtube.js

```diff
@@ -15,8 +15,8 @@ template.innerHTML = `
 <style>
   :host {
     display: inline-block;
-    --primary-color: #eee;
-    --secondary-color: transparent;
+    --media-secondary-color: transparent;
```

Comment:
Another example of usage.

Reply:
just wanted to make a note here that if a select-menu/listbox was used, the text would become unreadable in most cases.